jx_sqlite/aggs_table.py

```diff
@@ -360,12 +360,21 @@ def _get(row):
                     column_number = len(outer_selects)
 
                     for json_type, sql in details.sql.items():
-                        concat_sql.append("GROUP_CONCAT(QUOTE(DISTINCT(" + sql + ")))")
+                        concat_sql.append("GROUP_CONCAT(DISTINCT(" + sql + "))")
```

Comment:
`NULL`'s are still included. I think using where clause to not including them may be a bad logic.